ruleset/rules/0800-sysmon_id_1.xml

```diff
@@ -1,300 +1,271 @@
-    <rule id="92130" level="14">
-        <if_group>sysmon_event1</if_group>
-        <field name="win.eventdata.originalFileName" type="pcre2">(?i)reg.EXE</field>
-        <field name="win.eventdata.commandLine" type="pcre2">(?i)save.+\\\\SAM</field>
-        <description>Reg.exe used to dump SAM hive</description>
-        <mitre>
-            <id>T1003.002</id>
-        </mitre>
-    </rule>
+<!-- 
+  Sysmon Event ID 1 rules           
+-->
 
+<group name="sysmon_eid1_detections,">
 
-</group>
\ No newline at end of file
+  <rule id="92010" level="4">
+    <if_group>sysmon_event1</if_group>
+    <field name="win.eventdata.parentImage" type="pcre2">(?i)\\(c|w)script\.exe</field>
+    <description>Scripting interpreter spawned a new process.</description>
+    <mitre>
+      <id>T1059.005</id>
+    </mitre>
+  </rule>
+
+  <rule id="92011" level="6">
+    <if_sid>92010</if_sid>
+    <field name="win.eventdata.commandLine" type="pcre2">(?i)\\(c|w)script\.exe.+\.(bat|cmd|lnk|pif|vbs|vbe|js|wsh|ps1)</field>
+    <description>Scripting interpreter spawned new scripting interpreter.</description>
+    <mitre>
+      <id>T1059</id>
+    </mitre>
+  </rule>
+
+  <rule id="92012" level="6">
+    <if_sid>92010</if_sid>
+    <field name="win.eventdata.commandLine" type="pcre2">\\cmd\.exe</field>
+    <description>Scripting interpreter spawned Windows command shell instance.</description>
+    <mitre>
+      <id>T1059.003</id>
+    </mitre>
+  </rule>
+
+  <rule id="92013" level="6">
+    <if_sid>92010</if_sid>
+    <field name="win.eventdata.image" type="pcre2">(?i)[c-z]:(\\\\Users\\\\Public\\\\|\\\\Users\\\\.+\\\\AppData\\\\local\\\\temp)</field>
+    <description>Scripting interpreter spawned a process from a suspicious path.</description>
+    <mitre>
+      <id>T1059</id>
+    </mitre>
+  </rule>
+
+  <rule id="92021" level="4">
```

Comment:
If we need to have those ID jumps, we should justify them.

Reply:
Done